.changeset/spicy-bobcats-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@logto/next": minor
```

Comment:
Should this be a patch release?

Reply:
New api function added, so I think it is a minor release